from convert import filevk
list=["vk","we","kg","good"]
print(list[3]) #IndexError: list index out of range
res=filevk("Shubham") #Exception will occur same as it occured in convert.py.#It imported file from convert.py
                      
print(res)